unit_tests/test_utils_data_validator.py

```diff
@@ -0,0 +1,94 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2022 ScyllaDB
+import logging
+import os
+import unittest
+
+from sdcm.utils.data_validator import LongevityDataValidator
+from sdcm import sct_config
+
+
+class ConfigurationBase(unittest.TestCase):  # pylint: disable=too-many-public-methods
+    _testMethodName = "runTest"
+
+    @classmethod
+    def setUpClass(cls):
+        logging.basicConfig(level=logging.ERROR)
```

Comment:
why are you configuring the logging in the test itself ?
use the pytest `--log-cli-level=debug` for debugging of the test.

anyhow if we want to disable/enable specific loggers, we should do it in one central place. like in a fixture or hooks in `conftest.py`

Reply:
here's a commit, with same test but in pytest style: https://github.com/scylladb/scylla-cluster-tests/commit/ba290a78cead792e98dd72225e1ff43313f1687f

Reply:
changed